doc/misc/filtering.7.rst

```diff
@@ -28,42 +25,49 @@
 Description
 ===========
 
-Package filtering filters packages out from the available package set, making them invisible to most
-of the ``DNF5`` commands. They cannot be used in a transaction. Packages can be filtered out by either
-`Exclude Filtering` or `Modular Filtering`.
+Package filtering removes packages from the available package set, making them invisible to most ``DNF5`` commands. As a result, these packages cannot be included in any transaction.
 
+There are several ways a package can be filtered out:
 
-Exclude Filtering
-=================
+Global excludes filtering
+=========================
+
+Exclude filtering can be adjusted using the ``includepkgs`` or ``excludepkgs`` configuration options in the DNF5 configuration file. To disable excludes, you can use the ``disable_excludes`` configuration option:
+
+``dnf5 --setopt=disable_excludes=* install bash``
+
+For details about the configuration options see :manpage:`dnf5.conf(5)`, :ref:`DNF5 configuration reference <dnf5_conf-label>`.
+
+
+Repository excludes filtering
+=============================
 
-Exclude Filtering is a mechanism used by a user or by the ``DNF5`` plugin to modify the set of available
-packages. Exclude Filtering can be modified by either ``includepkgs`` or ``excludepkgs`` configuration options in
-configuration files. In addition to user-configured excludes, plugins can also extend the set of excluded packages.
-To disable excludes from the ``DNF5`` plugin you can use the ``--disable-plugin`` command line option.
+Similar to global excludes, but this configuration is repository-specific and only affects packages within the repository where the excludes are set.
 
-To disable all excludes for e.g. the install command you can use the following combination
-of command line options:
 
-``dnf5 --disableexcludes=all --disable-plugin="*" install bash``
+User excludes filtering
+=======================
 
+API users have an additional option how to fine-tune excluded packages using ``*_user_excludes()`` and ``*_user_includes`` methods of the ``PackageSack`` object. See :ref:`PackageSack API <api_packagesack-label>`.
 
-Modular Filtering
+
+Versionlock
+===========
+
+Additionally, the versionlock functionality is implemented using excludes filtering. However, these excludes are applied only during transactional operations.
+For details see :manpage:`dnf5-versionlock(8)`, :ref:`Versionlock command <versionlock_command_ref-label>`.
+
+
+Modular filtering
 =================
 
 Please see `the modularity documentation` for details on how Modular Filtering works.
```

Comment:
I cannot find `the modularity documentation`. If the documentation does not exist, consider removing this sentence.

Reply:
Oh, right. The documentation exists, so let me revise it quickly and add to the package.